app/src/main/java/se/inera/intyg/minaintyg/util/html/CertificateQuestionValueHTMLFactory.java

```diff
@@ -21,4 +27,16 @@ public static String table(CertificateQuestionValueTable question) {
   public static String list(CertificateQuestionValueList question) {
     return HTMLListFactory.ul(question.getValues());
   }
+
+  public static String itemList(CertificateQuestionValueItemList question) {
+    return p(
+        fromList(
+            question.getValues(),
+            valueItem -> join(
+                h5(valueItem.getLabel()),
```

Comment:
I think it's supposed to be h4 according to figma

Reply:
Fixed!